Cargo.toml

```diff
@@ -56,3 +56,12 @@ default-features = false
 
 [profile.release]
 debug = 2
```

Comment:
Can we tweak this to = 1 which should be enough for good backtraces and see if that solves most of the size problem?

Reply:
Still a bit bigger (~61 mb) than using the profile in this patch (~4 mb).
```
$ la -h target/{release,debug1}/{compiler,types,lang,prioritization-agenda}
-rwxr-xr-x 2 me me  61M Jan 25 11:39 target/debug1/compiler
-rwxr-xr-x 2 me me  62M Jan 25 11:39 target/debug1/lang
-rwxr-xr-x 2 me me  62M Jan 25 11:39 target/debug1/prioritization-agenda
-rwxr-xr-x 2 me me  61M Jan 25 11:39 target/debug1/types
-rwxr-xr-x 2 me me 140M Jan 25 11:36 target/release/compiler
-rwxr-xr-x 2 me me 141M Jan 25 11:36 target/release/lang
-rwxr-xr-x 2 me me 141M Jan 25 11:36 target/release/prioritization-agenda
-rwxr-xr-x 2 me me 140M Jan 25 11:36 target/release/types
```

The second biggest "contributor" in the profile in this patch is `strip = true` that brings the size from ~61mb down to ~6mb so probably:
```
[profile.release]
debug = 1
strip = true
```
would bring home most of the benefits.

Reply:
Anyway: no problem, I'll just keep this patch in a stash for personal use.

Thanks for your time.